### PR TITLE
share button updated

### LINKED ISSUE #1493: App sharing via Facebook
**Actual Behaviour**

Toast saying "Please retry again" is raised when I try to share the app using "INVITE VIA FACEBOOK" button. 
On the other hand, I am able to share the app successfully via Facebook using "SHARE APP" button and then selecting "share with Facebook".

**Expected Behaviour**

I think there is no need for a separate button to enable Facebook sharing. Moreover, it is not even functional.

**Screenshots of the issue**

device-2017-05-06-170435

device-2017-05-06-170458

**Would you like to work on the issue?**
Yes.

-----
@the-dagger @mariobehling @mananwason Is this intended?

### PR DESCRIPTION
**Pull request by mahikaw**
Fixes #1493
Removed "sharing with Facebook" button and restructured "sharing with app".

### CODE REVIEW

**Comment by mahikaw** [comment]
@the-dagger Please review it now.

**Comment by iamareebjamal** [comment]
@mahikaw Please resolve conflicts in MainActivity

**Comment by the-dagger** [comment]
@mahikaw Looks like there was an issue with resolving the conflicts, the travis build is failing,

**Comment by mahikaw** [comment]
@the-dagger please review now.

File: android/app/src/main/res/layout/dialog_share.xml [hunk 1]
```diff
@@ -1,40 +1,18 @@
...
+    android:layout_height="350dp">
 
     <Button
         android:id="@+id/share_app"
-        android:layout_width="245dp"
-        android:layout_height="wrap_content"
-        android:layout_gravity="center_horizontal"
-        android:layout_marginTop="30dp"
+        android:layout_width="match_parent"
+        android:layout_height="50dp"
```
Comment (iamareebjamal) [inline android/app/src/main/res/layout/dialog_share.xml]: Was `wrap_content` giving issues. Hardcoding values is bad IMO

**Comment by mahikaw** [comment]
@iamareebjamal fixed :)

**Pull request merged**